src/Model/Anime/Anime.php

```diff
@@ -207,6 +207,11 @@ class Anime
      */
     private $externalLinks = [];
 
+    /**
+     * @var Url[]
+     */
+    private $streamingLinks = [];
```

Comment:
I'm going to make the same observation as on @nerg4l 's PR #444. if we target php 8.0, we should use type names for member variables. In this case it would be: `private array $streamingLinks = [];`

Reply:
Sounds good. Yes, PHP 8.0 should be targeted 👍